lib/MusicBrainz/Server/Data/URL.pm

```diff
@@ -20,6 +20,7 @@ my %URL_SPECIALIZATIONS = (
     # External links section
     '45cat'               => qr{^https?://(?:www\.)?45cat\.com/}i,
     '45worlds'            => qr{^https?://(?:www\.)?45worlds\.com/}i,
+    'ACUM'                => qr{^https?://nocs.acum.org.il/acumsitesearchdb/}i,
```

Comment:
It feels like skipping acumsitesearchdb might be fine, but you might know better. You should escape the dots so they match just dots and not anything.

Reply:
done